Sources/SentryCrash/Recording/SentryCrashReport.c

```diff
@@ -1701,6 +1702,16 @@ sentrycrashreport_writeStandardReport(
         } else {
             writer->beginObject(writer, SentryCrashField_User);
         }
+
+        char *scopeJSON;
+        sentryscopesync_getJSON(&scopeJSON);
```

Comment:
Using this as out-parameter is super weird. why not just return the allocation?
Also, usually you shouldn’t allocate in the crash handler. It would seem that the `writer` has all kinds of methods to stich json objects together. Might be a good idea to use those instead of messing with `strcat` and friends.